src/javascript/app_2/App/Components/Layout/Header/__tests__/upgrade_button.spec.jsx

```diff
@@ -0,0 +1,14 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import { UpgradeButton }      from '../upgrade_button.jsx';
+
+configure({ adapter: new Adapter() });
+
+describe('UpgradeButton', () => {
+    it('should render one <UpgradeButton /> component', () => {
+        const wrapper = shallow(<UpgradeButton />);
+        expect(wrapper).to.have.length(1);
+    });
```

Comment:
Please add a test case for `onclick` prop as well as assertion that it will be called on click.

Reply:
I see we are many situations like this, maybe this functionality can also be extracted into a helper test function like `componentShouldHaveClickableProperty(component_name)` that accepts the name of the component (or the component it self and performs the following: 
1. Check if component has prop `onclick` 
2. Check if prop `onclick` is of type `function`
3. Check if Clicking on the component will result the prop to be called. 

Of course I'm just fantasizing this idea. that is totally up to you to do it or not :)

Reply:
That's a good idea but Most of the onClicks are defined in parent and therefore they are not defined or change anything in the unit directly